ccd-definition/CaseEventToComplexTypes/DefendantResponseLRspec.json

```diff
@@ -48,6 +48,94 @@
     "DisplayContext": "OPTIONAL",
     "FieldShowCondition": "respondent1=\"DO_NOT_SHOW_IN_UI\""
   },
+  {
+    "ID": "Debts",
+    "CaseEventID": "DEFENDANT_RESPONSE_SPEC",
```

Comment:
If DisplayContext MANDATORY works, please remove these entries from file

Reply:
Removed these entries after use of MANDATORY DisplayContext in CaseEventToFields/DefendantResponseAdmitPartLRspec.json